### ISSUE TITLE
为什么用 onnx 导出的 softmax 层会被分解成多个层，而不是单个 softmax 层

### ISSUE THREAD

**Issue by ItchyHiker** [Tencent/ncnn#1795]
自己benchmark了下发现分解为多个层，要比只使用softmax层实现速度慢一些

**Comment by nihui** [comment]
@daquexian  AT 大老师

**Comment by nihui** [comment]
https://github.com/onnx/onnx/issues/2289
这里在讨论

**Comment by nihui** [comment]
针对onnx模型转换的各种问题，推荐使用最新的pnnx工具转换到ncnn
In view of various problems in onnx model conversion, it is recommended to use the latest pnnx tool to convert your model to ncnn

```shell
pip install pnnx
pnnx model.onnx inputshape=[1,3,224,224]
```

详细参考文档
Detailed reference documentation
https://github.com/pnnx/pnnx
https://github.com/Tencent/ncnn/wiki/use-ncnn-with-pytorch-or-onnx#how-to-use-pnnx
